_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -48,7 +48,7 @@ Using `aria-hidden="false"` on a descendant of an element with `aria-hidden="tru
 
 By adding `aria-hidden="true"` to an element, content authors ensure that assistive technologies will ignore the element. This can be used to hide parts of a web page that are [pure decoration](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration), such as icon fonts - that are not meant to be read by assistive technologies.
 
-A [focusable][] element with `aria-hidden="true"` is ignored as part of the reading order, but still part of the focus order, making its state of [visible](#visible) or hidden unclear.
+However, if the element is also part of the [sequential focus navigation][], the assistive technology will want to announce the element where focus has landed. If focus lands on an element with `aria-hidden="true"`, the element's state of decorative vs non-decorative and [visible](#visible) vs hidden unclear.
```

Comment:
Is decorative/non-decorative a state? 

Also, end of last sentence should be "is unclear" - a word is missing.

Reply:
Also need to define 

> [sequential focus navigation][] 

at the bottom of the file

Reply:
@HelenBurge I see that already defined in this PR - it was missing in the other PR. I copied the definition from the other definitions - did I do something wrong there?

Reply:
@kengdoj "state" may be the wrong word, but I'm not sure of a better one offhand. `aria-hidden` is intended to indicate that the content is decorative. If it's focusable, it's interactive and therefore implied that it's not decorative. Would it be better to say "If an element with `aria-hidden="true"` can be tabbed to, it is unclear if that content is intended to be decorative or not"?

Reply:
@HelenBurge Disregard. I think you meant the other two definitions I missed. I'll get those.

Reply:
How about:
An element with an aria-hidden attribute set to true that is also part of the [sequential focus navigation][] may cause confusion for users of assistive technologies because the element should be hidden and not [included in the accessibility tree][].

This is the [definition of included in the accessibility tree](https://act-rules.github.io/glossary/#included-in-the-accessibility-tree): 

> [Programmatically hidden](https://act-rules.github.io/glossary/#programmatically-hidden) elements are removed from the accessibility tree. However, some browsers will leave [focusable](https://act-rules.github.io/glossary/#focusable) elements with an aria-hidden attribute set to true in the accessibility tree. Because they are hidden, these elements are considered not included in the accessibility tree. This may cause confusion for users of assistive technologies because they may still be able to interact with these focusable elements using sequential keyboard navigation, even though the element should not be included in the accessibility tree.

You can see the discussion about the wording of this in https://github.com/act-rules/act-rules.github.io/pull/1684

Reply:
@kengdoj Does this work?
> An element with an `aria-hidden` attribute set to `true` that is also part of the [sequential focus navigation][] may cause confusion for users of assistive technologies because the element can be reached via [sequential focus navigation][], but it should be hidden and not [included in the accessibility tree][].

Reply:
yes!